### DISCUSSION TITLE
How to handle multiple messages with same transaction_id and action

### DISCUSSION THREAD

**Discussion by tarung** [ONDC-Official/mobility-specification#24] [General]
If multiple messages are received with same transaction_id and action should we consider it to be duplicate and discard the message ?
Or these cases need to be handled differently based on the action

For example :
I received two select messages, with different item ids but same transaction_id should we consider that that two items were selected ?
I received two search request with same transaction_id should we discard the old search query and reinitiate a new one ?

**Comment by 92shreyansh** [comment]
for same transaction_id and action if the message_id is different then those are 2 different messages. 
In case of the context being exactly identical you can consider the request as duplicate. Fundamentally the context should be different for each request
